tor_core/__init__.py

```diff
@@ -1,5 +1,5 @@
 import os
 
-__version__ = '0.4.3'
+__version__ = '0.4.4'
```

Comment:
This seems like a feature addition, not a bugfix. Perhaps `0.5.0` would be more appropriate?